docs/source/releases/latest/627-CLI-tree.yaml

```diff
@@ -0,0 +1,83 @@
+enhancement:
+- description: |
+    *From GEOIPS#627: 2024-05-31, Add functionality to the CLI which prints out a tree of available commands*
+
+    The GeoIPS CLI provides a variety of commands which aren't necessarily easily exposed
+    via ``geoips -h``. To improve this issue, we've added a ``geoips tree`` command which
+    exposes all GeoIPS CLI commands in a tree-like fashion. This way, we can expose all
+    commands that are available via the GeoIPS CLI, and expose the depth in which these
+    commands exist.
+
+    By displaying the commands in a depthwise structure, users can understand what commands
+    are available and how they are called.
+
+    If you just call ``geoips tree``, you'll get the full command tree in a non-colored,
+    verbose output.
+
+    The output of running ``geoips tree`` is shown below.
+
+    .. code-block:: bash
+
+        geoips tree
+
+        geoips
+            geoips config
+                geoips config install
+            geoips get
+                geoips get family
+                geoips get interface
+                geoips get package
+                geoips get plugin
+            geoips list
+                geoips list interface
```

Comment:
Note this is different now.

Reply:
Ah; yes it is. Is it worth it to change this in this PR or should we do one final swoop at the CLI documentation once the other CLI PRs are merged. Since this is going to have to be hardcoded, it might just be best to wait fixing this until all new commands are merged. Up to you though.

Reply:
Let's update the docs at the end of getting the CLI merged.

Reply:
Works for me